root/release/edit/recordings.tt

```diff
@@ -128,6 +128,12 @@
           [% l('Copy all track artist credits to associated recordings.') %]
         </label>
       </p>
+      <p>
+        [% l('Only check these boxes if the recording titles/artists listed above are incorrect and you want to replace them with the track titles/artists) for this release.') %]
+      </p>
+      <p>
+        [% l('💡<strong>Note:</strong> In classical music, {csg_recording_artist|recording artists} should almost always be different from {csg_track_artist|track artists}, so don’t use the “copy artist credits” option while editing a classical release unless you’re very sure it’s a good idea!', {csg_recording_artist => doc_link('Style/Classical/Recording/Artist'), csg_track_artist => doc_link('Style/Classical/Track/Artist')}) %]
```

Comment:
1. 💡 should belong to style, not to message;
2. If we plan to add more notes like this, a dedicated TT macro/React component using a message '{alert}: {content}' would probably be helpful.

Reply:
I'm not sure what "{alert}: {content}" would give the translator to be honest. It'd allow you to change the order of the alert and content messages, or change the colon, but that's already possible here. There's also less context if you detach the messages from each other.

Reply:
So the suggestion would be putting💡out of the string and into a note_icon() macro or something like that? Or literally add it via CSS?

Reply:
@mwiencek: The benefit of having `<strong>Note:</strong>` apart is that it doesn’t have to be translated 20 times when there are 20 notes. Context can be added with `lp`. But there is probably not 20 notes right now, so this might be more pain for little help at the moment.

@reosarevok: Probably both.

Reply:
I understand that, but I meant the "{alert}: {content}" string specifically. :) If you have separate strings already, it seems to me that you can just display them next to each other rather than interpolate them together like that.

Reply:
Good catch, that string is erroneous anyway.